lib/exponent-server-sdk.rb

```diff
@@ -2,6 +2,36 @@
 require 'typhoeus'
 require 'json'
 
+
+# Basic Usage:
+#
+# Create new client
+# client = Exponent::Push::Client.new(**args)
+#
+# Send UPTO ~~100~~ messages per request,
+# Use [].each_slice(100) to split your data or it will fail
```

Comment:
Is there a reason this is a concern of the consumer of the API? Couldn't `send_messages` check if the length is > 100 and batch the messages for the consumer? If there is a reason to leave it up the consumer to batch the messages, you might want to add a warning check in send messages if > 100 are passed.

Reply:
@audiolion good point. Do you think it should just warn and ignore them? or fail entirely?

Reply:
not sure what is best

Reply:
I added a check for now that raises an error if you attempt to call it with more than 100